### PR TITLE
New deposit UI

### PR DESCRIPTION
**Pull request by Tougee**

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -847,7 +847,8 @@
     <string name="withdrawal_address">提现地址</string>
     <string name="withdrawal_no_tag">没有 Memo（备注）</string>
     <string name="deposit_account_attention">注意：首次充值建议小额尝试！地址 和 Memo（备注）同时使用才能充值 %1$s 到 Mixin Messenger。</string>
-    <string name="deposit_memo_notice">地址 和 Memo（备注）同时使用才能充值 %1$s 到你的账号。</string>
+    <string name="deposit_notice">地址 和 Memo（备注）同时使用才能充值 %1$s 到你的账号。</string>
+    <string name="deposit_memo_notice">充值时务必填写 Memo(备注)，否则您会丢失您的数字币</string>
```
Comment (SeniorZhai) [inline app/src/main/res/values-zh-rCN/strings.xml:851]: Lost period ?

**Review by SeniorZhai — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_deposit_key.xml [hunk 2]
```diff
@@ -4,108 +4,104 @@
...
-        android:textSize="13sp"/>
+                    <one.mixin.android.widget.BadgeCircleImageView
+                        android:id="@+id/qr_avatar"
+                        android:layout_width="36dp"
+                        android:layout_height="36dp"
+                        android:layout_gravity="center" />
+                </FrameLayout>
+            </androidx.constraintlayout.widget.ConstraintLayout>
+
+            <TextView
```
Comment (SeniorZhai) [inline app/src/main/res/layout/fragment_deposit_key.xml:94]: No margin bottom?

Comment (Tougee) [reply app/src/main/res/layout/fragment_deposit_key.xml:94]: QR image has a large margin around it self, so no more margin needed.

File: app/src/main/res/values-zh-rCN/strings.xml (hunk 1 above)
Comment (Tougee) [reply app/src/main/res/values-zh-rCN/strings.xml:851]: rename `deposit_memo_notice` to `deposit_notice`, and `deposit_memo_notice` is a new string

**Pull request merged**